pkg/plugins/manager/installer.go

```diff
@@ -156,7 +160,7 @@ func (m *PluginInstaller) Remove(ctx context.Context, pluginID string) error {
 		}
 	}
 
-	return nil
+	return m.serviceRegistry.RemoveExternalService(ctx, pluginID)
```

Comment:
mostly for readability now but I think it's better to check here if there is an associated service. Something like:

```go
if ok := m.serviceRegistry.HasExternalService(ctx, pluginID); ok {
 // delete it
 }
```